src/librustdoc/lib.rs

```diff
@@ -187,6 +189,7 @@ pub fn opts() -> Vec<getopts::OptGroup> {
         optopt("e", "extend-css",
                "to redefine some css rules with a given file to generate doc with your \
                 own theme", "PATH"),
+        optmulti("Z", "", "enable unstable options (only on nightly build)", "FLAG"),
```

Comment:
Could the text here be "internal and debugging options" to more closely match rustc? `-Z` isn't just for the `unstable-options` flag (in theory).